packages/evm/contracts/interfaces/ICommitteeCoordinator.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.8.26;
+
+interface ICommitteeCoordinator {
```

Comment:
It seems like this should perhaps be called `IRegistryFilter`?

Reply:
Sounds good. This was in any case, just a placeholder name, I was counting in advance on your input for the rename